src/memory/client/MemoryBrowser.tsx

```diff
@@ -77,21 +77,52 @@ interface State {
   bytesPerRow: number;
   bytesPerGroup: number;
   endianness: 'le' | 'be';
+  childNames: { id: string; name: string }[];
```

Comment:
Can you change `childNames` to `childrenNames` and similar changes elsewhere in the code.

Reply:
I have updated it